### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/management.md`

### PR DESCRIPTION
**Pull request by nahyunkeem**

### CODE REVIEW

**Comment by sodaMelon** [comment]
idk why the new docs are not being created by github action 


hi @tiangolo, could you check this problem again? 
https://github.com/fastapi/fastapi/pull/12527 (you said it was fixed recently, but.........  🤔 )

File: docs/ko/docs/management.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## FastAPI 전문가
+
+GitHub 토론에서 다른 사람들을 가장 많이 도와주는 사람은 [FastAPI 전문가]{.internal-link target=_blank}가 될 수 있습니다.
+이는 일반적으로 프로젝트에 기여하는 가장 좋은 방법입니다.
+
+## 외부 기여
+
+질문에 대한 답변, PR 제출 등 외부 기여를 매우 환영하고 감사하게 생각합니다. 🙇‍♂️
+[FastAPI를 유지하는 데 도움을 줄 수 있는](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/help-fastapi.md#help-maintain-fastapi){.internal-link target=_blank}[] 방법은 여러 가지가 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/management.md:33]:
```suggestion
[FastAPI를 유지하는 데 도움을 줄 수 있는](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/help-fastapi.md#help-maintain-fastapi){.internal-link target=_blank} 방법은 여러 가지가 있습니다.
```
This is unnecessary.

**Review by sodaMelon — COMMENTED** [review]
LGTM.
For now, the documentation is not being generated. so we'll have to wait a bit longer.

**Comment by alejsdev** [comment]
Hi @nahyunkeem :nerd_face: 
Thanks for your interest in contributing to the project! However, we're not translating this page (that's why the Build Docs is failing), you can see more information here: https://7fd003ac.fastapitiangolo.pages.dev/contributing/#dont-translate-these-pages
So, I'll close this one, but thanks for the effort! :rocket:

**Pull request closed without merging**